cookiecutter/exceptions.py

```diff
@@ -1,103 +1,101 @@
 # -*- coding: utf-8 -*-
 
-"""
-cookiecutter.exceptions
------------------------
-
-All exceptions used in the Cookiecutter code base are defined here.
-"""
+"""All exceptions used in the Cookiecutter code base are defined here."""
 
 
 class CookiecutterException(Exception):
-    """
-    Base exception class. All Cookiecutter-specific exceptions should subclass
-    this class.
+    """Base exception class.
+
+    All Cookiecutter-specific exceptions should subclass this class.
     """
 
 
 class NonTemplatedInputDirException(CookiecutterException):
-    """
-    Raised when a project's input dir is not templated.
+    """Raised when a project's input dir is not templated.
+
     The name of the input directory should always contain a string that is
     rendered to something else, so that input_dir != output_dir.
     """
 
 
 class UnknownTemplateDirException(CookiecutterException):
-    """
-    Raised when Cookiecutter cannot determine which directory is the project
-    template, e.g. more than one dir appears to be a template dir.
+    """Raised when Cookiecutter cannot determine template directory.
```

Comment:
Please keep the original wording.

Reply:
I changed it so that the full description is kept as it was initially, but for the summary it is not as easy as it has to fit in one line (and one line budget is 80 chars according to pep8).
So I kept my version here.
If this doesn't match your intent, please let me know how would you like this phrased. 

As a side note, it's worth mentioning that this exception does no seem to be used anywhere in the code (I grepped for it both in the current master and in the history).